kqueen/utils/logger_config.yml

```diff
@@ -43,3 +51,8 @@ loggers:
         level: INFO
         handlers: [error_file_handler, general_file_handler]
         propogate: yes
+
+    werkzeug:
```

Comment:
werkzeug logger does not used anywhere, where do u prefer to summon it?

Reply:
It is internal logger and it can be useful to track requests to the server. It looks like 

2018-03-14 10:51:40,478 - _internal - werkzeug - INFO: 127.0.0.1 - - [14/Mar/2018 10:51:40] "GET /ui/login HTTP/1.1" 200 -
2018-03-14 10:51:40,685 - _internal - werkzeug - INFO: 127.0.0.1 - - [14/Mar/2018 10:51:40] "GET /static/favicon/favicon-32x32.png?v1.0=3eepn6WlLO HTTP/1.1" 200 -
2018-03-14 10:51:42,380 - _internal - werkzeug - INFO: 127.0.0.1 - - [14/Mar/2018 10:51:42] "POST /ui/login HTTP/1.1" 302 -

it is defined here https://github.com/pallets/werkzeug/blob/60670cb98db1934b7aa7df454cbe4508c467f403/werkzeug/_internal.py#L112